src/client_for_pysim.cpp

```diff
@@ -70,7 +70,7 @@ int main (int argc, char* argv[]) {
             auto image_data = data.array["image.data"].as<uint16_t>();
             for (auto& v : image_data) { assert(v >= 1500 && v <= 1600); }
 
-            auto ptr = reinterpret_cast<const uint16_t*>(data.array["image.data"].data());
+            auto ptr = data.array["image.data"].data<uint16_t>();
```

Comment:
I think that by default the simulation now sends float32. Maybe we can change it here to make sure the example does not break.

Reply:
data = np.zeros(array_shape, dtype=np.uint16) # np.float32)
Do you mean you are going to change that or it is now in another branch which will be merged later?